README.md

```diff
@@ -1,13 +1,13 @@
 # Koji Custom VCC SDK
 ![npm (scoped)](https://img.shields.io/npm/v/@withkoji/custom-vcc-sdk?color=green&style=flat-square)
 
-**Library for creating custom controls to use in Koji templates.**
+**Library for creating custom controls to use in Koji app.**
 
 ## Overview
 
-The @withkoji/custom-vcc-sdk package enables you to implement custom controls for capturing user input on the frontend of your Koji template.
-With this package, you can provide customizations that match closely with the application you are developing.
-For example, some Koji templates provide tile map editors, sound enhancers, or custom avatar creators to enhance the interactivity for remixers.
+The @withkoji/custom-vcc-sdk package enables you to implement custom controls for capturing user input on the frontend of your Koji app.
+With this package, you can provide customizations that is better suited for the application you are developing.
+For example, some Koji apps provide tile map editors, sound enhancers, or custom avatar creators to enhance the interactivity in the app.
 
 ## Installation
 
```

Comment:
Did we want to remove `--save` from the installation comand?

Reply:
Fixed.